app/models/organization.rb

```diff
@@ -270,14 +273,17 @@ def suspended?
     is_suspended?
   end
 
-  def generate_access_token
-    begin
-      self.access_token = SecureRandom.hex
-    end while self.class.exists?(access_token: access_token)
+  def update_associations
+    parent_organization&.update_attributes(updated_at: Time.now)
   end
 
   private
 
+  def calculated_paid_feature_slugs
+    slugs = current_invoices.feature_slugs
+    child_ids.any? ? slugs + ["child_organizations"] : slugs
```

Comment:
Doing this because we're going to be checking if they have child organizations on every organized page render, and I'm concerned about performance.